### PR TITLE
Add Doc: Create a Multi-branch Pipeline with GitLab

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

File: content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+2. At the same time, a new tag has been generated in GitLab.
+
+   ![gitlab-result](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/gitlab-result.png)
+
+3. The sample application will be deployed to `kubesphere-sample-dev` and `kubesphere-sample-prod` with corresponding Deployments and Services created.
+
+   | Environment | URL                         | Namespace              | Deployment    | Service       |
+   | ----------- | --------------------------- | ---------------------- | ------------- | ------------- |
+   | Development | `http://{NodeIP}:{$30861}`  | kubesphere-sample-dev  | ks-sample-dev | ks-sample-dev |
```
Comment (JohnNiang) [inline content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md:147]: Missing `$` sign before `NodeIP`.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md (hunk 1 above)
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md]: Thanks for your comment. I added `$` here and also did the same thing in the doc Create a Pipeline Using a Jenkinsfile.

Comment (JohnNiang) [reply content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md]: Awesome!

File: content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+   ![create-pipeline](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/create-pipeline.png)
+
+3. In the **GitLab** tab, select the default option `https://gitlab.com` for GitLab Server, enter the username of the GitLab project owner for **Owner**, and then select the `devops-java-sample` repository from the drop-down list for **Repository Name**. Click the tick icon in the bottom right corner and then click **Next**.
+
+   ![select-gitlab](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/select-gitlab.png)
+
+   {{< notice note >}}
+
+   If you want to use a private repository from GitLab, you need to create an access token with API and read_repository permissions on GitLab, create a credential for accessing GitLab on Jenkins dashboard, and then add the credential in **GitLab Server** under **Configure System**. For more information about how to log in to Jenkins, refer to [Jenkins System Settings](../how-to-integrate/sonarqube/#step-5-add-the-sonarqube-server-to-jenkins).
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md:93]: on Jenkins dashboard
->
on the Jenkins dashboard

File: content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+   ![select-branch](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/select-branch.png)
+
+   {{< notice note >}}
+
+   The pipeline pauses at the stage `deploy to dev`. You need to click **Proceed** manually. Note that the pipeline will be reviewed three times as `deploy to dev`, `push with tag`, and `deploy to production` are defined in the Jenkinsfile respectively.
+
+   {{</ notice >}}
+
+### Step 6: Check pipeline status
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md:123]: ->
Check the pipeline status

**Pull request merged**